### PR TITLE
Add rubocop configuration

### PR DESCRIPTION
**Pull request by jhanggi**
I’m thinking we should just run rubocop as we’re working on new files; I don’t think it’s worth a big push on its own.

If we’re going to be doing rubocop cleanup, I’d like to see it always be its own commit when possible to make it easier to see what’s actually changes (especially in small PRs).

### CODE REVIEW

File: .rubocop-txi.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+AllCops:
+  Exclude:
+    - "db/schema.rb" # You can't touch this
+    - ".bundle/**/*" # Auto-generated
+    - "bin/**/*"     # Auto-generated
+    - "vendor/**/*"  # We cannot solve the world's problems
```
Comment (drench) [inline .rubocop-txi.yml]: For this project, I think we want to include `vendor/**/*`.

**Comment by drench** [comment]
We should probably add `rubocop` to the `Gemfile` as well.

File: .rubocop-txi.yml (hunk 1 above)
Comment (jhanggi) [inline .rubocop-txi.yml]: Good call. I'll do that.

**Comment by jhanggi** [comment]
Updated. (I was thinking about running rubocop over Gemfile, but wanted to keep the changes readable).

**Comment by drench** [comment]
:+1:

**Pull request closed without merging**